lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -69,6 +85,7 @@ def __init__(
         store: SqliteStore,
         subscriber_page_size: int,
         sync_interval: int,
+        suciprofile_db_dict: dict,
```

Comment:
```suggestion
        suciprofile_db_dict: Dict,
```
this should be coming from `from typing import Dict`

Reply:
Resolved as suggested.